app/javascript/src/components/Dashboard/Notes/Note.jsx

```diff
@@ -44,7 +44,6 @@ const Note = ({ note }) => (
           user={{
             imageUrl: "https://xsgames.co/randomusers/avatar.php?g=pixel",
           }}
-          onClick={function noRefCheck() {}}
```

Comment:
Check the browser console and see if there are any warnings. I believe these components should be having an onClick mandatorily. But if that's not the case, then no issues.

Refer: https://academy.bigbinary.com/learn-react/pages/prefer-es6-arrow-functions#:~:text=Always%20prefer%20arrow%20functions

Reply:
There is a warning in my console, which even after adding `onClick={() => {}}` in the Avatar component persists. I am attaching the screenshot of the warning. However there is no such warnings related to the `onClick` function.
<img width="1437" alt="Screenshot 2022-09-20 at 7 49 55 PM" src="https://user-images.githubusercontent.com/43422638/191282916-38d3062c-92d7-44cf-9af4-462e66d09a98.png">

Reply:
You can ignore that warning.

Reply:
Thank you Yedhin.